doc/specs/stdlib_bitsets.md

```diff
@@ -0,0 +1,2006 @@
+
+#### Class
+
+Subroutine
+
+#### Arguments
+
+`self`: shall be a scalar variable of class `bitset_64` or
+`bitset_large`. It is an `intent(out)` argument.
+
+`unit`: shall be a scalar default integer expression. It is an
+`intent(in)` argument. Its value must be that of a logical unit
+number for an open unformatted file with `READ` or `READWRITE`
+access positioned at the start of a BITSET value written by a
+`bitset_type` `output` subroutine by the same processor.
+
+`status` (optional): shall be a scalar default integer variable. If
+present its value shall be of one of the error codes defined in this
+module. IF absent and it would have had a value other than `success`
+processing will stop with an informative stop code. Allowed error code
+values for this `status` are:
+
+* `success` - no problem found
+
+* `alloc_fault` - `self` was of type `bitset_large` and allocation of
+  memory failed.
+
+* `array_size_invalid_error` - if the number of bits read from `unit`
+  is either negative or greater than 64, if class of `self` is
+  `bitset_64`.
+
+* `read_failure` - failure during a read statement
+
+#### Example
+
+```fortran
+    program demo_input
+        character(*), parameter :: &
+            bits_0   = '000000000000000000000000000000000', &
+            bits_1   = '000000000000000000000000000000001', &
+            bits_33  = '100000000000000000000000000000000'
+        integer :: unit
+        type(bitset_64) :: set0, set1, set2, set3, set4, set5
+        call set0 % from_string( bits_0 )
+        call set1 % from_string( bits_1 )
+        call set2 % from_string( bits_33 )
+        open( newunit=unit, file='test.bin', status='replace', &
+            form='unformatted', action='write' )
+        call set2 % output(unit)
+        call set1 % output(unit)
+        call set0 % output(unit)
+        close( unit )
+        open( newunit=unit, file='test.bin', status='old', &
+            form='unformatted', action='read' )
+        call set5 % input(unit)
+        call set4 % input(unit)
+        call set3 % input(unit)
+        close( unit )
+        if ( set3 /= set0 .or. set4 /= set1 .or. set5 /= set2 ) then
+            error stop 'Transfer to and from units using ' // &
+                ' output and input failed.'
+        else
+            write(*,*) 'Transfer to and from units using ' // &
+                'output and input succeeded.'
+        end if
+    end program demo_input
+```
+
+### `none` - determines whether no bits are set
+
+#### Status
+
+Experimental
+
+#### Description
+
+Determines whether no bits are set in `self`.
+
+#### Syntax
+
+`result = self % [[bitset_type(class):none(bound)]] ()`
+
+#### Class
+
+Elemental function.
+
+#### Argument
+
+`self`: shall be a scalar expression of class `bitset_type`. It is an
+  `intent(in)` argument.
+
+#### Result character
+
+The result is a default logical scalar.
+
+#### Result value
+
+The result is `.true.` if no bits in `self` are set, otherwise it is
+`.false.`.
```

Comment:
```suggestion
#### Result value

The result is a default logical scalar.
The result is `.true.` if no bits in `self` are set, otherwise it is
`.false.`.
```